PROCESS.md

```diff
@@ -0,0 +1,54 @@
+# Documentation process for Kubermatic projects
+
+This document describes the documentation process for Kubermatic projects.
+
+## What must be documented?
+
+All user-facing changes must be documented. Documentation for KKP,
+KubeOne, and KubeCarrier is located in this repository. Documentation for
+other projects is usually located in the respective repository.
+
+## How are changes documented?
+
+Changes are documented by creating a PR in the appropriate repository. The PR
```

Comment:
Isn't it always docs repository?

Reply:
@csengerszabo For KKP, KubeOne, and KubeCarrier, but not for other projects. For example, machine-controller has its docs here: https://github.com/kubermatic/machine-controller/tree/main/docs